src/Couchbase.Lite.Shared/API/Database/DatabaseConfiguration.cs

```diff
@@ -69,29 +69,30 @@ public string Directory
         /// </remarks>
         internal bool EnableVersionVector => false;
 
-        #if COUCHBASE_ENTERPRISE
-        /// <summary>
-        /// Gets or sets the encryption key to use on the database
-        /// </summary>
-        [CanBeNull]
-        public EncryptionKey EncryptionKey { get; init; }
-        #endif
-
         #endregion
 
+        #region Constructors
+
         /// <summary>
         /// Default constructor
         /// </summary>
-        public DatabaseConfiguration()
+        /// <param name="directory">
+        /// Default directory is <see cref="Service.GetRequiredInstance<IDefaultDirectoryResolver>().DefaultDirectory()" /> if directory set to null.
+        /// </param>
+        public DatabaseConfiguration(
+#if COUCHBASE_ENTERPRISE
+            EncryptionKey encryptionKey = null,
```

Comment:
Let's have this as the last argument instead of the first so that if the user says `new DatabaseConfiguration(directory)` it will work in both CE and EE.